apps/core/components/userContext/SignInHelpers.web.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+import firebase from 'firebase/app';
+
+export const signInWithGoogle = async () => {
+  const provider = new firebase.auth.GoogleAuthProvider();
```

Comment:
Is localization taken into account here? I don't know if that's is something that has to be taken into account or we just use the default. To use the device locale we could also set it with `firebase.auth().useDeviceLanguage()`